ui/admin/app/components/mapping-list/index.hbs

```diff
@@ -0,0 +1,90 @@
+<Hds::Form::Fieldset
+  @layout={{@layout}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  as |F|
+>
+  {{yield F}}
+  <F.Control>
+    <Hds::Table class='mapping-table' name={{@name}}>
+      <:head as |H|>
+        <H.Tr>
+          <H.Th>{{t 'form.key.label'}}</H.Th>
+          <H.Th>{{t 'form.value.label'}}</H.Th>
+          <H.Th>{{t 'titles.actions'}}</H.Th>
+        </H.Tr>
+      </:head>
+      <:body as |B|>
+        {{#each @options as |option index|}}
+          <B.Tr>
+            <B.Td>
+              <Hds::Form::TextInput::Field
+                @value={{option.key}}
+                @type='text'
+                disabled={{@disabled}}
```

Comment:
Non blocking comment, asking from curiosity, wher this `@disabled` come from? tracked prop?

Reply:
It's getting passed in by the consumer, usually from the form's determination of `disabled`